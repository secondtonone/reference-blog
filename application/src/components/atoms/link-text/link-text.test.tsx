import renderer from 'react-test-renderer';
import LinkText from '.';

describe('Match snapshot LinkText', () => {
  it('match snapshot default', () => {
    const tree = renderer.create(<LinkText href="#">link</LinkText>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
