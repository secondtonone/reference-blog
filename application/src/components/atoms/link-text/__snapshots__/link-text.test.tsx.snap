// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot LinkText match snapshot default 1`] = `
<a
  className="link-text__StyledLink-sc-a7posi-0 kCvSzJ SLink"
  data-test="link-text"
  href="#"
>
  link
</a>
`;
